app/templates/base.html

```diff
@@ -36,3 +36,5 @@
 {% endblock %}
 {% block footer %}
 {% endblock %}
+{% block extend_js %}
```

Comment:
we have js block in base template.

Reply:
https://bootstrap-flask.readthedocs.io/en/latest/basic.html?highlight=javascript#starter-template